extensions/trace-propagators/src/test/java/io/opentelemetry/extension/trace/propagation/OtTracerPropagatorTest.java

```diff
@@ -114,6 +115,20 @@ void inject_NotSampledContext() {
     assertThat(carrier).containsEntry(OtTracerPropagator.SAMPLED_HEADER, "false");
   }
 
+  @Test
```

Comment:
Have an additional test, to make sure that upon no valid `SpanContext` in `Context`, the `Bagagge` **was not** injected? Likewise for extraction ;)

Reply:
added